todos/src/Components/TodosList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
```

Comment:
About OptionBar (it doesnt allow me to write comments there for some reason).
Extract the type of an alias/interface.